src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -22,19 +35,24 @@
           type="text"
           className="input"
           placeholder="Search..."
+          value={filter.query}
```

Comment:
Make sure that `filter.query` is initialized as a string to avoid potential runtime errors when setting the input value.